build.xml

```diff
@@ -133,6 +133,20 @@
     <mkdir dir="build"/>
   </target>
 
+  <property name="ivy.install.version" value="2.3.0" />
+  <condition property="ivy.home" value="${env.IVY_HOME}">
```

Comment:
If we utilize the `IVY_HOME` then I think we run into two problems. One is that `download-ivy` will try to overwrite the jar in it. The other is that the jar in `IVY_HOME` is certainly not part of Ant's classpath, so it won't be able to use the ivy tasks. So I guess we just shouldn't use `IVY_HOME`.